src/main/java/org/gridsuite/modification/server/dto/byfilter/equipmentfield/GeneratorField.java

```diff
@@ -98,7 +98,8 @@ public static void setNewValue(Generator generator, String generatorField, @NotN
                 ActivePowerControl<Generator> activePowerControl = generator.getExtension(ActivePowerControl.class);
                 ActivePowerControlAdder<Generator> activePowerControlAdder = generator.newExtension(ActivePowerControlAdder.class);
                 ModificationUtils.getInstance().modifyActivePowerControlAttributes(activePowerControl, activePowerControlAdder, null,
-                        new AttributeModification<>(Float.parseFloat(newValue), OperationType.SET), null, null);
+                        new AttributeModification<>(Float.parseFloat(newValue), OperationType.SET), null, null,
+                    MODIFY_GENERATOR_ERROR, "Generator '" + generator.getId() + "' : ");
```

Comment:
you can create variable errorMessage for "Generator '" + generator.getId() + "' : "